### ISSUE TITLE
Failing imports 

### ISSUE THREAD

**Issue by djhoese** [conda-forge/eccodes-feedstock#120]
- [ ] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

I have some CI that just started failing last night due to not being able to import python-eccodes. However, I'm unable to reproduce the issue locally so I'm out of ideas for how to debug this. Here is a failing CI job on GitHub:

https://github.com/pytroll/satpy/pull/1563/checks?check_run_id=1957708029

```
     try:
>       import eccodes as ec

satpy/readers/ascat_l2_soilmoisture_bufr.py:31: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    from __future__ import absolute_import
    import sys
    
>   from .eccodes import *

/usr/share/miniconda3/envs/test-environment/lib/python3.8/site-packages/eccodes/__init__.py:15: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   from gribapi import __version__

/usr/share/miniconda3/envs/test-environment/lib/python3.8/site-packages/eccodes/eccodes.py:12: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   from .gribapi import *  # noqa

/usr/share/miniconda3/envs/test-environment/lib/python3.8/site-packages/gribapi/__init__.py:13: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    """
    @package gribapi
    @brief This package is the \b Python 3 interface to ecCodes. It offers almost one to one bindings to the C API functions.
    
    The Python 3 interface to ecCodes uses the <a href="http://numpy.scipy.org/"><b>NumPy</b></a> package
    as the container of choice for the possible arrays of values that can be encoded/decoded in and from a message.
    Numpy is a package used for scientific computing in Python and an efficient container for generic data.
    
    @em Requirements:
    
        - Python 3.5 or higher
        - NumPy
    
    """
    
    from functools import wraps
    import sys
    import os
    import numpy as np
    
>   from .bindings import ENC, ffi, lib

/usr/share/miniconda3/envs/test-environment/lib/python3.8/site-packages/gribapi/gribapi.py:32: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    from __future__ import absolute_import, division, print_function, unicode_literals
    
    import logging
    import pkgutil
    import os
    
    import cffi
    
    __version__ = "1.1.0"
    
    LOG = logging.getLogger(__name__)
    
    try:
>       from ._bindings import ffi, lib
E       ImportError: /usr/share/miniconda3/envs/test-environment/lib/python3.8/site-packages/gribapi/_bindings.cpython-38-x86_64-linux-gnu.so: undefined symbol: codes_bufr_key_is_header

/usr/share/miniconda3/envs/test-environment/lib/python3.8/site-packages/gribapi/bindings.py:29: ImportError
```

<br/>
Environment (<code>conda list</code>):
<details>

Conda create/install log is all I can get below.

Environment file here: https://github.com/pytroll/satpy/blob/master/continuous_integration/environment.yaml

```
  Package                             Version  Build                   Channel                     Size
  ─────────────────────────────────────────────────────────────────────────────────────────────────────────
    Install:
  ─────────────────────────────────────────────────────────────────────────────────────────────────────────
  
    affine                                2.3.0  py_0                    conda-forge/noarch         16 KB
    alabaster                            0.7.12  py_0                    conda-forge/noarch         15 KB
    appdirs                               1.4.4  pyh9f0ad1d_0            conda-forge/noarch         13 KB
    argon2-cffi                          20.1.0  py38h497a2fe_2          conda-forge/linux-64       48 KB
    asciitree                             0.3.3  py_2                    conda-forge/noarch          6 KB
    async_generator                        1.10  py_0                    conda-forge/noarch         18 KB
    attrs                                20.3.0  pyhd3deb0d_0            conda-forge/noarch         41 KB
    babel                                 2.9.0  pyhd3deb0d_0            conda-forge/noarch          6 MB
    backcall                              0.2.0  pyh9f0ad1d_0            conda-forge/noarch         13 KB
    backports                               1.0  py_2                    conda-forge/noarch          4 KB
    backports.functools_lru_cache         1.6.1  py_0                    conda-forge/noarch          8 KB
    behave                                1.2.6  py38h578d9bd_1003       conda-forge/linux-64      185 KB
    blas                                   2.14  openblas                conda-forge/linux-64       10 KB
    bleach                                3.3.0  pyh44b312d_0            conda-forge/noarch        111 KB
    bokeh                                 2.2.3  py38h578d9bd_0          conda-forge/linux-64        7 MB
    boost-cpp                            1.72.0  h9d3c048_4              conda-forge/linux-64       16 MB
    brotlipy                              0.7.0  py38h497a2fe_1001       conda-forge/linux-64      341 KB
    bzip2                                 1.0.8  h7f98852_4              conda-forge/linux-64      Cached
    c-ares                               1.17.1  h36c2ea0_0              conda-forge/linux-64      Cached
    cached-property                       1.5.2  py_0                    pkgs/main/noarch           11 KB
    cairo                                1.16.0  h7979940_1007           conda-forge/linux-64        1 MB
    cartopy                              0.18.0  py38h2516ec2_10         conda-forge/linux-64        2 MB
    cffi                                 1.14.5  py38ha65f79e_0          conda-forge/linux-64      227 KB
    cfitsio                               3.470  hce51eda_7              conda-forge/linux-64        1 MB
    cftime                                1.4.1  py38h5c078b8_0          conda-forge/linux-64      325 KB
    chardet                               4.0.0  py38h578d9bd_1          conda-forge/linux-64      199 KB
    click                                 7.1.2  pyh9f0ad1d_0            conda-forge/noarch         64 KB
    click-plugins                         1.1.1  py_0                    conda-forge/noarch          9 KB
    cligj                                 0.7.1  pyhd8ed1ab_0            conda-forge/noarch         10 KB
    cloudpickle                           1.6.0  py_0                    conda-forge/noarch         22 KB
    codecov                              2.1.11  pyhd3deb0d_0            conda-forge/noarch         20 KB
    colorama                              0.4.4  pyh9f0ad1d_0            conda-forge/noarch         18 KB
    colorcet                              2.0.6  pyhd8ed1ab_0            conda-forge/noarch          1 MB
    configobj                             5.0.6  py_0                    conda-forge/noarch         31 KB
    coverage                                5.4  py38h497a2fe_0          conda-forge/linux-64      265 KB
    coveralls                             3.0.0  pyhd8ed1ab_0            conda-forge/noarch         16 KB
    cryptography                          3.4.4  py38h3e25421_0          conda-forge/linux-64      637 KB
    curl                                 7.71.1  he644dc0_8              conda-forge/linux-64      139 KB
    cycler                               0.10.0  py_2                    conda-forge/noarch          9 KB
    cython                              0.29.22  py38h709712a_0          conda-forge/linux-64        2 MB
    cytoolz                              0.11.0  py38h497a2fe_3          conda-forge/linux-64      420 KB
    dask                               2021.2.0  pyhd8ed1ab_0            conda-forge/noarch          4 KB
    dask-core                          2021.2.0  pyhd8ed1ab_0            conda-forge/noarch        681 KB
    datashader                           0.12.0  pyhd3deb0d_0            conda-forge/noarch         14 MB
    datashape                             0.5.4  py_1                    conda-forge/noarch         49 KB
    dbus                                1.13.18  hb2f20db_0              pkgs/main/linux-64        504 KB
    decorator                             4.4.2  py_0                    conda-forge/noarch         11 KB
    defusedxml                            0.6.0  py_0                    conda-forge/noarch         22 KB
    distributed                        2021.2.0  py38h578d9bd_0          conda-forge/linux-64        1 MB
    docopt                                0.6.2  py_1                    conda-forge/noarch         14 KB
    docutils                               0.16  py38h578d9bd_3          conda-forge/linux-64      738 KB
    donfig                                0.6.0  pyhd8ed1ab_0            conda-forge/noarch         16 KB
    eccodes                              2.19.1  hea64003_0              conda-forge/linux-64        4 MB
    entrypoints                             0.3  py38h32f6830_1002       conda-forge/linux-64       13 KB
    expat                                2.2.10  h9c3ff4c_0              conda-forge/linux-64      164 KB
    fasteners                              0.16  pyhd3eb1b0_0            pkgs/main/noarch           24 KB
    fiona                                1.8.18  py38h37fbd03_0          conda-forge/linux-64        1 MB
    fontconfig                           2.13.1  hba837de_1004           conda-forge/linux-64      344 KB
    freetype                             2.10.4  h0708190_1              conda-forge/linux-64      890 KB
    freexl                                1.0.6  h27cfd23_0              pkgs/main/linux-64         42 KB
    fsspec                                0.8.5  pyhd8ed1ab_0            conda-forge/noarch         69 KB
    gdal                                  3.1.4  py38h25844d8_6          conda-forge/linux-64        1 MB
    geopandas                             0.8.2  pyhd8ed1ab_0            conda-forge/noarch        933 KB
    geos                                  3.9.0  h9c3ff4c_0              conda-forge/linux-64        1 MB
    geotiff                               1.6.0  h11d48b3_4              conda-forge/linux-64      278 KB
    geoviews                              1.8.2  py_0                    conda-forge/noarch          6 KB
    geoviews-core                         1.8.2  py_0                    conda-forge/noarch        402 KB
    gettext                            0.19.8.1  h0b5b191_1005           conda-forge/linux-64        4 MB
    giflib                                5.2.1  h516909a_2              conda-forge/linux-64       80 KB
    glib                                 2.66.7  h9c3ff4c_0              conda-forge/linux-64      441 KB
    glib-tools                           2.66.7  h9c3ff4c_0              conda-forge/linux-64       85 KB
    gst-plugins-base                     1.18.3  h04508c2_0              conda-forge/linux-64        2 MB
    gstreamer                            1.18.3  h3560a44_0              conda-forge/linux-64        2 MB
    h5netcdf                             0.10.0  pyhd8ed1ab_0            conda-forge/noarch         18 KB
    h5py                                  3.1.0  nompi_py38hafa665b_100  conda-forge/linux-64        1 MB
    hdf4                                 4.2.13  h10796ff_1004           conda-forge/linux-64      951 KB
    hdf5                                 1.10.6  nompi_h7c3c948_1111     conda-forge/linux-64        3 MB
    heapdict                              1.0.1  py_0                    conda-forge/noarch          7 KB
    holoviews                            1.14.1  pyhd3deb0d_0            conda-forge/noarch          3 MB
    icu                                    68.1  h58526e2_0              conda-forge/linux-64      Cached
    idna                                   2.10  pyh9f0ad1d_0            conda-forge/noarch         52 KB
    imageio                               2.9.0  py_0                    conda-forge/noarch          3 MB
    imagesize                             1.2.0  py_0                    conda-forge/noarch          8 KB
    importlib-metadata                    3.4.0  py38h578d9bd_0          conda-forge/linux-64       20 KB
    importlib_metadata                    3.4.0  hd8ed1ab_0              conda-forge/noarch          3 KB
    iniconfig                             1.1.1  pyh9f0ad1d_0            conda-forge/noarch          8 KB
    ipykernel                             5.5.0  py38h81c977d_1          conda-forge/linux-64      167 KB
    ipython                              7.20.0  py38h81c977d_2          conda-forge/linux-64        1 MB
    ipython_genutils                      0.2.0  py_1                    conda-forge/noarch         21 KB
    jasper                              1.900.1  h07fcdf6_1006           conda-forge/linux-64      286 KB
    jedi                                 0.18.0  py38h578d9bd_2          conda-forge/linux-64      937 KB
    jinja2                               2.11.3  pyh44b312d_0            conda-forge/noarch         93 KB
    jpeg                                     9d  h516909a_0              conda-forge/linux-64      266 KB
    json-c                                 0.15  h98cffda_0              conda-forge/linux-64      274 KB
    jsonschema                            3.2.0  py38h32f6830_1          conda-forge/linux-64       90 KB
    jupyter_client                       6.1.11  pyhd8ed1ab_1            conda-forge/noarch         76 KB
    jupyter_core                          4.7.1  py38h578d9bd_0          conda-forge/linux-64       72 KB
    jupyterlab_pygments                   0.1.2  pyh9f0ad1d_0            conda-forge/noarch          8 KB
    kealib                               1.4.14  he4dc956_1              conda-forge/linux-64      186 KB
    kiwisolver                            1.3.1  py38h1fd1430_1          conda-forge/linux-64       80 KB
    krb5                                 1.17.2  h926e7f8_0              conda-forge/linux-64      Cached
    lcms2                                  2.12  hddcbb42_0              conda-forge/linux-64      443 KB
    libaec                                1.0.4  he1b5a44_1              conda-forge/linux-64       32 KB
    libblas                               3.8.0  14_openblas             conda-forge/linux-64       10 KB
    libcblas                              3.8.0  14_openblas             conda-forge/linux-64       10 KB
    libclang                             11.0.1  default_ha53f305_1      conda-forge/linux-64       19 MB
    libcurl                              7.71.1  hcdd3856_8              conda-forge/linux-64      Cached
    libdap4                              3.20.6  hd7c4107_1              conda-forge/linux-64       11 MB
    libedit                        3.1.20191231  he28a2e2_2              conda-forge/linux-64      121 KB
    libev                                  4.33  h516909a_1              conda-forge/linux-64      Cached
    libevent                             2.1.10  hcdb4288_3              conda-forge/linux-64        1 MB
    libgdal                               3.1.4  hbde00c8_6              conda-forge/linux-64       13 MB
    libgfortran-ng                        7.5.0  h14aa051_18             conda-forge/linux-64       22 KB
    libgfortran4                          7.5.0  h14aa051_18             conda-forge/linux-64        1 MB
    libglib                              2.66.7  h1f3bc88_0              conda-forge/linux-64        3 MB
    libiconv                               1.16  h516909a_0              conda-forge/linux-64      Cached
    libkml                                1.3.0  hd79254b_1012           conda-forge/linux-64      640 KB
    liblapack                             3.8.0  14_openblas             conda-forge/linux-64       10 KB
    liblapacke                            3.8.0  14_openblas             conda-forge/linux-64       10 KB
    libllvm10                            10.0.1  he513fc3_3              conda-forge/linux-64       26 MB
    libllvm11                            11.0.1  hf817b99_0              conda-forge/linux-64       29 MB
    libnetcdf                             4.7.4  nompi_h56d31a8_107      conda-forge/linux-64        1 MB
    libnghttp2                           1.43.0  h812cca2_0              conda-forge/linux-64      Cached
    libopenblas                           0.3.7  h5ec1e0e_6              conda-forge/linux-64        8 MB
    libpng                               1.6.37  hed695b0_2              conda-forge/linux-64      359 KB
    libpq                                  13.1  hfd2b0eb_1              conda-forge/linux-64        3 MB
    librttopo                             1.1.0  hccdd1c9_5              conda-forge/linux-64      235 KB
    libsodium                            1.0.18  h516909a_1              conda-forge/linux-64      366 KB
    libspatialindex                       1.9.3  he1b5a44_3              conda-forge/linux-64        3 MB
    libspatialite                         5.0.1  h04c9dda_2              conda-forge/linux-64        4 MB
    libssh2                               1.9.0  hab1572f_5              conda-forge/linux-64      Cached
    libtiff                               4.2.0  hdc55705_0              conda-forge/linux-64      633 KB
    libuuid                              2.32.1  h14c3975_1000           conda-forge/linux-64       26 KB
    libwebp-base                          1.2.0  h7f98852_0              conda-forge/linux-64      808 KB
    libxcb                                 1.14  h7b6447c_0              pkgs/main/linux-64        505 KB
    libxkbcommon                          1.0.3  he3ba5ed_0              conda-forge/linux-64      581 KB
    libxml2                              2.9.10  h72842e0_3              conda-forge/linux-64      Cached
    llvmlite                             0.35.0  py38h4630a5e_1          conda-forge/linux-64      239 KB
    locket                                0.2.1  py38h06a4308_1          pkgs/main/linux-64         10 KB
    lz4-c                                 1.9.3  h9c3ff4c_0              conda-forge/linux-64      Cached
    markdown                              3.3.3  pyh9f0ad1d_0            conda-forge/noarch         66 KB
    markupsafe                            1.1.1  py38h497a2fe_3          conda-forge/linux-64       27 KB
    matplotlib                            3.3.4  py38h578d9bd_0          conda-forge/linux-64        6 KB
    matplotlib-base                       3.3.4  py38h0efea84_0          conda-forge/linux-64        7 MB
    mistune                               0.8.4  py38h497a2fe_1003       conda-forge/linux-64       54 KB
    mock                                  4.0.3  py38h578d9bd_1          conda-forge/linux-64       51 KB
    monotonic                               1.5  py_0                    conda-forge/noarch          9 KB
    more-itertools                        8.7.0  pyhd8ed1ab_0            conda-forge/noarch         39 KB
    msgpack-python                        1.0.2  py38h1fd1430_1          conda-forge/linux-64       92 KB
    multipledispatch                      0.6.0  py_0                    conda-forge/noarch         12 KB
    munch                                 2.5.0  py_0                    conda-forge/noarch         12 KB
    mysql-common                         8.0.23  ha770c72_1              conda-forge/linux-64        2 MB
    mysql-libs                           8.0.23  h935591d_1              conda-forge/linux-64        2 MB
    nbclient                              0.5.2  pyhd8ed1ab_0            conda-forge/noarch         60 KB
    nbconvert                             6.0.7  py38h578d9bd_3          conda-forge/linux-64      546 KB
    nbformat                              5.1.2  pyhd8ed1ab_1            conda-forge/noarch         66 KB
    nest-asyncio                          1.5.1  pyhd3eb1b0_0            pkgs/main/noarch           10 KB
    netcdf4                               1.5.6  nompi_py38h1cdf482_100  conda-forge/linux-64      555 KB
    notebook                              6.2.0  py38h578d9bd_0          conda-forge/linux-64        6 MB
    nspr                                   4.29  he1b5a44_1              conda-forge/linux-64      232 KB
    nss                                    3.62  hb5efdd6_0              conda-forge/linux-64        2 MB
    numba                                0.52.0  py38h51da96c_0          conda-forge/linux-64        4 MB
    numcodecs                             0.7.3  py38h709712a_0          conda-forge/linux-64     1000 KB
    numpy                                1.20.1  py38h18fd61f_0          conda-forge/linux-64        6 MB
    olefile                                0.46  pyh9f0ad1d_1            conda-forge/noarch         32 KB
    openjpeg                              2.4.0  hf7af979_0              conda-forge/linux-64      525 KB
    packaging                              20.9  pyh44b312d_0            conda-forge/noarch         35 KB
    pandas                                1.2.2  py38h51da96c_0          conda-forge/linux-64       12 MB
    pandoc                               2.11.4  h7f98852_0              conda-forge/linux-64       18 MB
    pandocfilters                         1.4.3  py38h06a4308_1          pkgs/main/linux-64         14 KB
    panel                                0.10.3  pyhd8ed1ab_0            conda-forge/noarch          6 MB
    param                                1.10.1  pyhd3deb0d_0            conda-forge/noarch         64 KB
    parse                                1.19.0  pyh44b312d_0            conda-forge/noarch         22 KB
    parse_type                            0.5.2  py38h578d9bd_3          conda-forge/linux-64       46 KB
    parso                                 0.8.1  pyhd8ed1ab_0            conda-forge/noarch         67 KB
    partd                                 1.1.0  py_0                    conda-forge/noarch         17 KB
    pcre                                   8.44  he1b5a44_0              conda-forge/linux-64      261 KB
    pexpect                               4.8.0  py38h32f6830_1          conda-forge/linux-64       80 KB
    pickleshare                           0.7.5  py38h32f6830_1002       conda-forge/linux-64       13 KB
    pillow                                8.1.0  py38ha0e1e83_2          conda-forge/linux-64      668 KB
    pixman                               0.40.0  h36c2ea0_0              conda-forge/linux-64      627 KB
    pluggy                               0.13.1  py38h578d9bd_4          conda-forge/linux-64       29 KB
    pooch                                 1.3.0  pyhd8ed1ab_0            conda-forge/noarch         40 KB
    poppler                              0.89.0  h2de54a5_5              conda-forge/linux-64       16 MB
    poppler-data                         0.4.10  0                       conda-forge/noarch          4 MB
    postgresql                             13.1  h6303168_1              conda-forge/linux-64        5 MB
    proj                                  7.2.0  h277dcde_2              conda-forge/linux-64        3 MB
    prometheus_client                     0.9.0  pyhd3deb0d_0            conda-forge/noarch         44 KB
    prompt-toolkit                       3.0.16  pyha770c72_0            conda-forge/noarch        244 KB
    psutil                                5.8.0  py38h497a2fe_1          conda-forge/linux-64      342 KB
    ptyprocess                            0.7.0  pyhd3deb0d_0            conda-forge/noarch         16 KB
    py                                   1.10.0  pyhd3deb0d_0            conda-forge/noarch         73 KB
    pycparser                              2.20  pyh9f0ad1d_2            conda-forge/noarch         94 KB
    pyct                                  0.4.8  py38_0                  pkgs/main/linux-64         28 KB
    pygments                              2.8.0  pyhd8ed1ab_0            conda-forge/noarch        736 KB
    pyhdf                                0.10.2  py38h8c074d7_2          conda-forge/linux-64      148 KB
    pykdtree                              1.3.4  py38h0b5ebd8_0          conda-forge/linux-64       72 KB
    pylibtiff                             0.4.2  py38hf155ad6_3          conda-forge/linux-64      221 KB
    pyopenssl                            20.0.1  pyhd8ed1ab_0            conda-forge/noarch         48 KB
    pyparsing                             2.4.7  pyh9f0ad1d_0            conda-forge/noarch         60 KB
    pyproj                          3.0.0.post1  py38h8df5ac0_0          conda-forge/linux-64      536 KB
    pyqt                                 5.12.3  py38h578d9bd_7          conda-forge/linux-64       21 KB
    pyqt-impl                            5.12.3  py38h7400c14_7          conda-forge/linux-64        6 MB
    pyqt5-sip                           4.19.18  py38h709712a_7          conda-forge/linux-64      310 KB
    pyqtchart                              5.12  py38h7400c14_7          conda-forge/linux-64      257 KB
    pyqtwebengine                        5.12.1  py38h7400c14_7          conda-forge/linux-64      175 KB
    pyresample                           1.17.0  py38h0ef3d22_0          conda-forge/linux-64        2 MB
    pyrsistent                           0.17.3  py38h497a2fe_2          conda-forge/linux-64       90 KB
    pyshp                                 2.1.3  pyh44b312d_0            conda-forge/noarch         36 KB
    pysocks                               1.7.1  py38h578d9bd_3          conda-forge/linux-64       27 KB
    pytest                                6.2.2  py38h578d9bd_0          conda-forge/linux-64      432 KB
    pytest-cov                           2.11.1  pyh44b312d_0            conda-forge/noarch         21 KB
    python-dateutil                       2.8.1  py_0                    conda-forge/noarch        220 KB
    python-eccodes                    2021.01.0  py38h5c078b8_0          conda-forge/linux-64       95 KB
    python-geotiepoints                   1.2.0  py38h8790de6_0          conda-forge/linux-64      151 KB
    pytz                                 2021.1  pyhd8ed1ab_0            conda-forge/noarch        239 KB
    pyviz_comms                           2.0.1  pyhd3deb0d_0            conda-forge/noarch        134 KB
    pyyaml                                5.4.1  py38h497a2fe_0          conda-forge/linux-64      192 KB
    pyzmq                                22.0.3  py38h3d7ac18_0          conda-forge/linux-64      533 KB
    qt                                   5.12.9  hda022c4_4              conda-forge/linux-64      100 MB
    rasterio                              1.2.0  py38h033aa8a_0          conda-forge/linux-64        8 MB
    requests                             2.25.1  pyhd3deb0d_0            conda-forge/noarch         51 KB
    rtree                                 0.9.7  py38h02d302b_1          conda-forge/linux-64       45 KB
    scipy                                 1.6.1  py38hf56f3a7_0          pkgs/main/linux-64         16 MB
    send2trash                            1.5.0  py_0                    conda-forge/noarch         12 KB
    shapely                               1.7.1  py38h604dc72_2          conda-forge/linux-64      443 KB
    six                                  1.15.0  pyh9f0ad1d_0            conda-forge/noarch         14 KB
    snowballstemmer                       2.1.0  pyhd8ed1ab_0            conda-forge/noarch         57 KB
    snuggs                                1.4.7  py_0                    conda-forge/noarch          8 KB
    sortedcontainers                      2.3.0  pyhd8ed1ab_0            conda-forge/noarch         26 KB
    sphinx                                3.5.1  pyhd8ed1ab_0            conda-forge/noarch          1 MB
    sphinxcontrib-applehelp               1.0.2  py_0                    conda-forge/noarch         28 KB
    sphinxcontrib-devhelp                 1.0.2  py_0                    conda-forge/noarch         22 KB
    sphinxcontrib-htmlhelp                1.0.3  py_0                    conda-forge/noarch         27 KB
    sphinxcontrib-jsmath                  1.0.1  py_0                    conda-forge/noarch          7 KB
    sphinxcontrib-qthelp                  1.0.3  py_0                    conda-forge/noarch         25 KB
    sphinxcontrib-serializinghtml         1.1.4  py_0                    conda-forge/noarch         24 KB
    tblib                                 1.7.0  py_0                    pkgs/main/noarch           16 KB
    terminado                             0.9.2  py38h578d9bd_0          conda-forge/linux-64       26 KB
    testpath                              0.4.4  py_0                    conda-forge/noarch         85 KB
    tiledb                                2.2.4  hb9a9e87_0              conda-forge/linux-64        3 MB
    toml                                 0.10.2  pyhd8ed1ab_0            conda-forge/noarch         18 KB
    toolz                                0.11.1  py_0                    conda-forge/noarch         46 KB
    tornado                                 6.1  py38h497a2fe_1          conda-forge/linux-64      649 KB
    tqdm                                 4.57.0  pyhd8ed1ab_0            conda-forge/noarch         76 KB
    traitlets                             5.0.5  py_0                    conda-forge/noarch         81 KB
    typing_extensions                   3.7.4.3  py_0                    conda-forge/noarch         25 KB
    tzcode                                2021a  h7f98852_0              conda-forge/linux-64      296 KB
    urllib3                              1.26.3  pyhd8ed1ab_0            conda-forge/noarch         99 KB
    wcwidth                               0.2.5  pyh9f0ad1d_2            conda-forge/noarch         33 KB
    webencodings                          0.5.1  py_1                    conda-forge/noarch         12 KB
    xarray                               0.16.2  pyhd8ed1ab_0            conda-forge/noarch        546 KB
    xerces-c                              3.2.3  h9d8b166_2              conda-forge/linux-64        2 MB
    xorg-kbproto                          1.0.7  h14c3975_1002           conda-forge/linux-64       26 KB
    xorg-libice                          1.0.10  h516909a_0              conda-forge/linux-64       57 KB
    xorg-libsm                            1.2.3  h84519dc_1000           conda-forge/linux-64       25 KB
    xorg-libx11                          1.6.12  h516909a_0              conda-forge/linux-64      917 KB
    xorg-libxext                          1.3.4  h516909a_0              conda-forge/linux-64       51 KB
    xorg-libxrender                      0.9.10  h516909a_1002           conda-forge/linux-64       31 KB
    xorg-renderproto                     0.11.1  h14c3975_1002           conda-forge/linux-64        8 KB
    xorg-xextproto                        7.3.0  h14c3975_1002           conda-forge/linux-64       27 KB
    xorg-xproto                          7.0.31  h14c3975_1007           conda-forge/linux-64       72 KB
    yaml                                  0.2.5  h516909a_0              conda-forge/linux-64       82 KB
    zarr                                  2.6.1  pyhd8ed1ab_0            conda-forge/noarch        100 KB
    zeromq                                4.3.4  h9c3ff4c_0              conda-forge/linux-64      352 KB
    zict                                  2.0.0  py_0                    conda-forge/noarch         10 KB
    zipp                                  3.4.0  py_0                    conda-forge/noarch         11 KB
    zstd                                  1.4.8  ha95c52a_1              conda-forge/linux-64      Cached
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
Run conda-incubator/setup-miniconda@v2
  with:
    miniconda-version: latest
    python-version: 3.8
    mamba-version: *
    channels: conda-forge,defaults
    environment-file: continuous_integration/environment.yaml
    activate-environment: test-environment
    auto-activate-base: true
    auto-update-conda: false
    remove-profiles: true
    architecture: x64
  env:
    PYTHON_VERSION: 3.8
    OS: ubuntu-latest
    UNSTABLE: false
    ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```
</details>

**Comment by kmuehlbauer** [comment]
@djhoese Looks like python-eccodes is build with `eccodes=2.16`  over at [python_eccodes-feedstock](https://github.com/conda-forge/python-eccodes-feedstock/blob/master/recipe/meta.yaml). Maybe something changed from 2.16 to 2.19.

**Comment by djhoese** [comment]
Thanks! I'll try some stuff out. I tried `python-eccodes-feedstock` in my URL and of course didn't find the feedstock so that's why I filed it here. I assumed they were both being built in this recipe.

**Comment by djhoese** [comment]
Interesting, looks like 2.19.1 fails on my system which had defaulted to 2.20.1. Now to figure out why CI picked up 2.19.1 instead of 2.20.

**Comment by djhoese** [comment]
I filed a bug on python-eccodes just in case and have forced the version of eccodes on my CI and things seem to be working now. I confirmed that in a build ~24 hours ago eccodes 2.20.0 was installed but something has changed that made the solver choose 2.19.1 now. I'm not sure why and I'm not sure why it is doing that choice on my CI (github incubator action) but not on my local version of conda. I should maybe check versions of conda.

Closing for now as I'm not really sure this is eccodes problem.
